domenetjenester/medlem/src/main/java/no/nav/foreldrepenger/domene/medlem/UtledVurderingsdatoerForMedlemskapTjeneste.java

```diff
@@ -80,6 +79,20 @@ public Set<LocalDate> finnVurderingsdatoer(BehandlingReferanse ref) {
         return datoer.stream().filter(d -> d.isAfter(ref.getUtledetSkjæringstidspunkt()) && ref.getUtledetMedlemsintervall().encloses(d)).collect(Collectors.toSet());
     }
 
+    public Set<LocalDate> finnVurderingsDatoerForutForStpEngangsstønad(BehandlingReferanse ref) {
+        var endringssjekker = FagsakYtelseTypeRef.Lookup.find(alleEndringssjekkere, ref.fagsakYtelseType())
+            .orElseThrow(() -> new IllegalStateException("Ingen implementasjoner funnet for ytelse: " + ref.fagsakYtelseType().getKode()));
+        var fom = ref.getUtledetSkjæringstidspunkt().minusMonths(12);
```

Comment:
UtledetSTP vil gi fødselsdato dersom denne finnes, ellers termin, eller omsorgsovertakelse.
Til vurderingen så trenger vi vel max(fødsel, termin) minus 12 mnd?

Reply:
> minst 12 måneder sammenhengende, direkte forut for tidspunktet for fødselsterminen
eller omsorgsovertakelsen.

Hva betyr "fødseltermin"? Jeg tolket det som det første utsagnet som du sier der: Dersom fødselsdato finnes bruk denne, ellers termin, ellers omsorgsovertakelse?

Reply:
Baselinedato for de 12 månedene må avklares i full detalj med PL - fødsel før/på/etter termin og uten fødsel

Reply:
Men det er ikke viktig for dette loggingsformålet

Reply:
Da forsetter vi med utledetSTP da i denne omgang?